client/knowledge_base_search/src/containers/SearchContainer/FeatureSelect.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import { string, func, element } from 'prop-types'
+import classNames from 'classnames';
+import './styles.css';
+
+class FeatureSelect extends Component {
+  // a map of the features shown in this starter kit
+  static featureTypes = {
+    // Passage search - finds relevant passages across all documents
+    PASSAGES: {
+      value: 'passages',
+      text: 'Passage Search'
+    },
+    // Relevancy training - uses user-input training data to improve relevancy
+    TRAINED: {
+      value: 'trained',
+      text: 'Relevancy'
+    }
+  }
+
+  isSelected(feature) {
+    return this.props.selectedFeature === feature.value;
+  }
+
+  render() {
+    const {
+      onFeatureSelect,
+      questionTypeSelector
+    } = this.props;
+
+    return (
+      <ul className="feature_select--list">
+        <li className="feature_select--list_item">
+          { questionTypeSelector }
+        </li>
+        {
+          Object.keys(FeatureSelect.featureTypes).map((featureKey) => {
```

Comment:
I haven't been blessed enough to use statics yet, but I'm wondering why you have to use `FeatureSelect.staticThingy` instead of `this.staticThingy`.

Reply:
#blessed